psydac/core/bsplines.py

```diff
@@ -656,23 +665,35 @@ def elevate_knots(knots, degree, periodic):
     periodic : bool
         True if domain is periodic, False otherwise.
 
+    multiplicity : int
+        multiplicity of the knots.
```

Comment:
See my previous comment to the `make_knots` function

Reply:
done